content/features/featuresDeepDive/Exporters/glTFExporter.md

```diff
@@ -150,4 +147,3 @@ Key:
 - Material Extensions
   - Draco Mesh Compression [Babylon.js #8046](https://github.com/BabylonJS/Babylon.js/issues/8046)
   - Translucency [KHR_Materials_Translucency](https://github.com/KhronosGroup/glTF/pull/1825) is still in draft
-- Camera Serialization [Babylon.js #9146](https://github.com/BabylonJS/Babylon.js/issues/9146)

```

Comment:
This issue has been closed for a while and the discussion indicates that the documentation is outdated, so that is why I marked camera export as supported and removed the link to the issue.